main.go

```diff
@@ -51,6 +51,8 @@ var (
 	logJson       = kingpin.Flag("json", "log in json format").Bool()
 	retryDelay    = kingpin.Flag("delay", "exponential retry - start delay time").Default(storclient.DefaultRetryDelay.String()).Duration()
 	retryAttempts = kingpin.Flag("attempts", "count of attempts of retry").Default(strconv.Itoa(storclient.DefaultRetryAttempts)).Uint()
+	extension     = kingpin.Flag("extension", "downloaded file extension - like '.dat' = SHA.dat").Default("").String()
```

Comment:
What about some check for '.' in extension? Otherwise I would call that variable suffix.

Reply:
extension is better name - thanks